indico/modules/events/registration/templates/display/regform_summary.html

```diff
@@ -1,14 +1,11 @@
-{% extends 'layout/conference_page_base.html' %}
-
-{% block title %}
-    {%- trans %}Registration{% endtrans -%}
-{% endblock %}
-
-{% block subtitle %}
-    {{ regform.title }}
-{% endblock %}
+{% extends 'events/registration/display/_regform_base.html' %}
```

Comment:
Could you wait until my changes get merged? I'm splitting this file in two different ones and it's gonna be quite messy to solve the conflict neatly.